cannon/mipsevm/state.go

```diff
@@ -2,11 +2,16 @@ package mipsevm
 
 import (
 	"encoding/binary"
+	"fmt"
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/crypto"
 )
 
+// StateWitnessSize is the size of the state witness encoding in bytes.
+var StateWitnessSize = 226
```

Comment:
Can we add a test to verify this is still accurate? Basically just encode a witness and assert the length == StateWitnessSize. Even if other tests would fail a specific test will make it a lot easier to understand why.